tutorials/basics/4_batch.py

```diff
@@ -250,12 +252,12 @@ def forward(self, g):
     (test_Y == argmax_Y.float()).sum().item() / len(test_Y) * 100))
 
 ###############################################################################
-# Below is an animation where we plot graphs with the probability a trained model
-# assigns its ground truth label to it:
+# The figure here is an animation where you plot graphs with the probability that a trained model
+# assigns its Amazon SageMaker ground truth label to it.
```

Comment:
As @yzh119 has explained, the label is not from Amazon SageMaker. It is a categorical variable from the dataset itself indicating the graph type of each sample. How about "The animation here plots the probability a trained model predicts the correct graph type."?